modules/monitoring/manifests/checks/rds_config.pp

```diff
@@ -23,8 +23,8 @@
   $region = undef,
   $cpu_warning = 80,
   $cpu_critical = 90,
-  $memory_warning = 20,
-  $memory_critical = 10,
+  $memory_warning = 2.0,
+  $memory_critical = 1.0,
```

Comment:
For consistency with the other values provided in this file, I think these should integers, so 2 and 1 here...